Gemfile.lock

```diff
@@ -165,9 +165,9 @@ GEM
       activesupport (>= 5.0.0)
       minitest
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.6.0)
+    rails-html-sanitizer (1.6.1)
       loofah (~> 2.21)
-      nokogiri (~> 1.14)
+      nokogiri (>= 1.15.7, != 1.16.7, != 1.16.6, != 1.16.5, != 1.16.4, != 1.16.3, != 1.16.2, != 1.16.1, != 1.16.0.rc1, != 1.16.0)
```

Comment:
This is an unusually long constraint set for the gem here. Wonder what's going on with bundle or the nokogiri repo that lead to that. Curious if we'll see the same thing on bundle update on other repos.